### ISSUE TITLE
Monetization via crypto mining

### ISSUE THREAD

**Issue by jtojnar** [hughsie/oars#16]
Recently a software mining cryptocurrencies [appeared in Snap store](https://blog.ubuntu.com/2018/05/15/trust-and-security-in-the-snap-store). Most people will probably agree cryptocurrency mining is not much worse than advertising, and that the main issue was that the action was not indicated to the user. As we already have advertising attribute, it might be useful to have “mining” or “resource utilization for monetary gain” in the Money category as well. It might have “None”, “Opt-out” and “Mandatory” options.

**Comment by hughsie** [comment]
I don't think this is rating the content, this is an attribute of how the software is packaged. You could package GIMP with a miner in theory. I also don't think mining would affect the age rating of the software, which is the typical thing we look at when adding categories to OARS.
